### PR TITLE
JAMES-2855 Rationalize subproject plugins / plugin management

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

File: mailbox/pom.xml [hunk 1]
```diff
@@ -82,35 +82,4 @@
...
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    </properties>
-
-    <repositories>
-        <!-- apache.snapshot is needed for james-project SNAPSHOT, see JAMES-1470 -->
-        <repository>
-            <releases>
-                <enabled>false</enabled>
-            </releases>
-            <id>apache.snapshots</id>
```
Comment (rouazana) [inline mailbox/pom.xml]: we don't need these for example if you rely temporary on a non released version of mime4j?

Comment (chibenwa) [reply mailbox/pom.xml]: > we don't need these for example if you rely temporary on a non released version of mime4j?

Why wouldn't we defined snapshot repositories in `james-project/pom.xml` ?

Having it at this level in the project hierarchy makes no sense...

**Comment by chibenwa** [comment]
`java.net.BindException: Address already in use`

test this please

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**